releases.yaml

```diff
@@ -1,9 +1,46 @@
 ---
 releases:
+  24.11:
+    releaseDate: &latest-release-release-date 2024-11-18
+    description: TBD
```

Comment:
@lfrancke, is there a one-liner description of the release that can go in here?

Reply:
I don't know any...where does that even show?

Reply:
In the `stckablectl release list` output.

Reply:
If anyone comes up with anything please add it.. Otherwise I'm fine with just a generic "The November 2024 release." As well.

Reply:
We should probably link to the release notes in future. But here is the suggestion to apply:

```suggestion
    description: The November 2024 release
```

I believe @Maleware is waiting on this before he can install the new operators during the ARM testing.